### PR TITLE
Add setup.py to enable instaling python-varnishapi via pip

### PR DESCRIPTION
**Pull request by ziollek**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    name='varnishapi',
+    version=release,
+    author='Shohei Tanaka(@xcir)',
+    author_email='',
+    description="Connect to libvarnish api by ctypes",
+    long_description=long_description,
+    install_requires=[],
+    keywords='varnish,varnishlog,varnishstat',
+    platforms=['any'],
+    license='Apache Software License v2.0',
```
Comment (xcir) [inline setup.py]: This library license is FreeBSD.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# -*- encoding: utf-8 -*-
+
+import os
+import sys
+from distutils.core import setup
+
+from pip.req import parse_requirements
+from setuptools import find_packages
+
+assert sys.version_info >= (3, 4), "Python 3.4+ required."
```
Comment (xcir) [inline setup.py]: Support 2.7.x and 3.4.x

**Comment by xcir** [comment]
Thank you for your p-r.
Would you this p-r send to pip branch?
Because, I want to consider some point.(versioning...)

**Comment by ziollek** [comment]
I fixed above problems and send p-r to branch pip

**Pull request closed without merging**